packages/ui/src/Select/MultiSelectField.tsx

```diff
@@ -165,6 +165,22 @@ export const MultiSelectField = <V extends string | number = number>(props: Mult
     }
   }, [menuIsOpen])
 
+  /** If the size of screen is changed we close dropdown menu to aviod the dropdown menu bug in react-select library.
```

Comment:
Cause we use the same functionality in Select it is better to move it into a custom hook.